source/scriptHandler.py

```diff
@@ -96,6 +98,19 @@ def findScript(gesture):
 		if func:
 			return func
 
+	# Braille display level
+	if isinstance(braille.handler.display, baseObject.ScriptableObject):
+		func = _getObjScript(braille.handler.display, gesture, globalMapScripts)
+		if func:
+			return func
+
+	# Vision enhancement provider level

```

Comment:
I can't find your initial comment about this, but I revisited this. Now, global plugins and appModules take precedence over vision enhancement provider scripts. I think this makes more sense than it was before.